src/common/store/reblogs/index.ts

```diff
@@ -79,6 +80,9 @@ export const fetchReblogs = () => (dispatch: Dispatch, getState: () => AppState)
 
     dispatch(fetchedAct(items));
   });
+
+  const announcementData = getAnnouncementsData(activeUser.username, 200);
```

Comment:
@MTSaeed Why adding this into reblogs store? Could create separate/dedicated one, no?